File: 백준/Silver/11399. ATM/ATM.py
import sys
readLine = sys.stdin.readline
n = int(readLine())
time = sorted(list(map(int, readLine().split())))
# print(time)
for i in range(1, len(time)):
    time[i] = time[i] + time[i - 1]
# print(time)
print(sum(time))

File: 백준/Silver/11399. ATM/ATM.swift
import Foundation

let n = Int(readLine()!)!
var member = readLine()!.split { $0 == " " }.map { Int($0)! }
member = member.sorted()
for i in 1..<member.count {
  member[i] = member[i] + member[i - 1]
}
var sums = member.reduce(0, +)
print(sums)
